Update README.md

Darthagnon opened the pull request:
Previous pull request didn't work?
https://github.com/Barre/privaxy/pull/22
Issue #21

Darthagnon committed:
Update README.md

Previous pull request didn't work?
https://github.com/Barre/privaxy/pull/22

Barre reviewed (changes requested):
Thank you for this PR. I'd prefer to not endorse directly in the README any particular filter set. Otherwise, looks ok to me.

Darthagnon commented:
> particular filter set

Is that the reference to filterlists.com?

Darthagnon committed:
Update README.md

Improve wording? filterlists.com is an aggregator, rather than a single list...?

Darthagnon commented:
Hmmm... that's a point - are custom filterlists currently supported?

ThinkChaos commented on README.md:
```diff
@@ -22,7 +22,7 @@ This is an early release without authentication on the management API. You shoul
 ### Using a pre-built binary
 
-Pre-built binaries for Macos and Linux (x86_64) are provided on [github releases](https://github.com/Barre/privaxy/releases).
+Pre-built binaries for MacOS, Linux (x86_64) and Windows are provided on [Github releases](https://github.com/Barre/privaxy/releases).
```
The (not so) new spelling is "macOS":
```suggestion
Pre-built binaries for macOS, Linux (x86_64) and Windows are provided on [Github releases](https://github.com/Barre/privaxy/releases).
```

ThinkChaos commented on README.md:
```diff
@@ -39,11 +39,15 @@ Pre-built binaries for Macos and Linux (x86_64) are provided on [github releases
 2. Install the downloaded certificate locally.
-    - Macos: <https://support.apple.com/guide/keychain-access/add-certificates-to-a-keychain-kyca2431/mac>
-    - Linux: `cp privaxy_ca_certificate.pem /usr/local/share/ca-certificates/`
+   - MacOS: https://support.apple.com/guide/keychain-access/add-certificates-to-a-keychain-kyca2431/mac
```
```suggestion
   - macOS: https://support.apple.com/guide/keychain-access/add-certificates-to-a-keychain-kyca2431/mac
```

ThinkChaos commented on README.md:
```diff
@@ -39,11 +39,15 @@ Pre-built binaries for Macos and Linux (x86_64) are provided on [github releases
 3. Configure your local system to pass http traffic through privaxy.
-   - Macos: <https://support.apple.com/guide/mac-help/change-proxy-settings-network-preferences-mac-mchlp2591/mac>
-   - Ubuntu (gnome): <https://phoenixnap.com/kb/ubuntu-proxy-settings>
+   - MacOS: https://support.apple.com/guide/mac-help/change-proxy-settings-network-preferences-mac-mchlp2591/mac
```
```suggestion
   - macOS: https://support.apple.com/guide/mac-help/change-proxy-settings-network-preferences-mac-mchlp2591/mac
```
